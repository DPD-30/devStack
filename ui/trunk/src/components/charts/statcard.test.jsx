import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesStatCard from './statcard.jsx'

jest.unmock('./statcard.jsx')


describe('', () => {
    it('PagesStatCard should match snapshot', () => {
        const comp = renderToString(<PagesStatCard isFail={false} isPass={false}/>)

        expect(comp).toMatchSnapshot()
    })
})
