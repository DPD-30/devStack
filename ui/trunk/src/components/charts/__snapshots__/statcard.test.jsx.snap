// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesStatCard should match snapshot 1`] = `
<div class="card"
     data-reactroot
>
  <div class="card-body">
    <div class="row">
      <div class="col-5">
        <div class="statIcon">
          <span>
          </span>
        </div>
      </div>
      <div class="col-7">
        <div class="statCard">
          <p class="statTitle">
          </p>
          <p class="statValue">
          </p>
        </div>
      </div>
    </div>
    <div class="statFooterBGColor card-footer">
      <div class="statFooterColor">
      </div>
    </div>
  </div>
</div>
`;
